features/282-next-repo-re-organization/plan.md

```diff
@@ -0,0 +1,63 @@
+# Re-organize Github Repo to focus on NEXT
+
+## Repository layout
+
+The top level is the git repository.
+
+- skygear
+    - gateway
+    - auth gear
+    - cloud functions and microservices
+- skycli
+- skygear-SDK-JS
+- skygear-SDK-iOS
+- skygear-SDK-Android
+- skygear-controller
+    - skycli and portal backend
+    - manage cluster users
+    - manage app tenant config
+    - CF controller (deploy cf to openfaas)
+- skygear-next-cloud
```

Comment:
Maybe don't call it `skygear-next-cloud`? Since NEXT is not the official name.

Reply:
We will keep this name, since this is a private repo. Original `skygear-cloud` are the v1 setup, we will update the repo description to avoid confusion.